wg/bsi.md

```diff
@@ -0,0 +1,58 @@
+# Cloud Native Batch System Initiative Working Group Charter
+ 
+## Introduction
+
+Cloud Native Batch System Initiative Working Group is a group formed by 
+passionate batch system maintainers looking to support batch workload in cloud native environments.
+ 
+## Background
+
+More and more users need to migrate batch workloads to cloud native environments,
+and multiple existing solutions from different community projects currently address this requirement,
+e.g. Volcano, YuniKorn, Armada, MCAD, which makes it easy for users to migrate.
+But from a user perspective, the current support for batch workloads is widely different from 
+one batch system to another and requires a comprehensive migration between them. 
+It’s necessary to build a specification for batch workloads in the cloud native ecosystem, 
+and help related projects figure out their scopes and interaction. The users and communities can 
+follow this specification to work with projects in CNCF, e.g. Kubernetes, Volcano, Armada, MCAD, 
+or even out of CNCF, e.g. YuniKorn, and so on.
+  
+## Goal:
+
+The Cloud Native Batch System Initiative Working Group aims to define
+batch system specifications in the cloud native ecosystem for different batch projects,
+and related CRDs, gRPC, client libraries for projects. Those outputs will help related projects
+to figure out their scopes.
+   
+## In Scope:
+
+Batch system specifications, including:
+
+- Job/ArrayJob/CronJob
+- Job Flow/Job Dependency/Job Orchestration
+- Queue
+- SchedulingPolicy (e.g. PodGroup)
+- Command Line & Script
+- Multi-Cluster Scheduling
+- Best Practices
+
+## Out Of Scope:
+
+- Non-batch related workloads
+- The implementation of the batch system
+    
+## Operations
+
+This Batch System Initiative Working Group follows the [standard operating guidelines](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#operating-model) provided
+by the TOC unless otherwise stated here.
+
+- TOC Liaisons: Ricardo Rocha (ricardo.rocha@cern.ch)
+- SIG Chairs: Klaus Ma (klaus1982.cn@gmail.com), Weiwei Yang (abvclouds@gmail.com), Alex Scammon (alex@gr-oss.io)
```

Comment:
How much is the WG targeting kubernetes?

(Note: I would consider volcano as targeting kubernetes, as it mainly runs in kubernetes and not other environments, like Mesos, to put an example)

The k8s Batch WG has an specific goal of "not reinventing the wheel". In other words, to improve the existing components and add extension points, rather than creating alternate implementations. This is crucial for cloud providers to be able to offer proper support to their users.

If you think this is also valuable in the context of this WG, I think it would be appropriate to have a chair with strong technical knowledge and leadership in the kubernetes project to serve as a steward for this goal.

Reply:
I'm open to have a chair from k8s :)